Formula/wget.rb

```diff
@@ -33,6 +34,8 @@ class Wget < Formula
   depends_on "libmetalink" => :optional
   depends_on "gpgme" => :optional
 
+  patch :p0, :DATA
```

Comment:
If the test fails, we need it reported upstream and use the upstream commit as a `patch`